static/app/components/events/interfaces/crashContent/stackTrace/contentV3.tsx

```diff
@@ -209,29 +209,45 @@ function Content({
     })
     .filter(frame => !!frame) as React.ReactElement[];
 
+  const className = `traceback ${
+    includeSystemFrames ? 'full-traceback' : 'in-app-traceback'
+  }`;
+
   if (convertedFrames.length > 0 && registers) {
     const lastFrame = convertedFrames.length - 1;
     convertedFrames[lastFrame] = cloneElement(convertedFrames[lastFrame], {
       registers,
     });
 
     return (
-      <Wrapper isHoverPreviewed={isHoverPreviewed} data-test-id="stack-trace">
-        {!newestFirst ? convertedFrames : [...convertedFrames].reverse()}
+      <Wrapper className={className}>
+        <Frames isHoverPreviewed={isHoverPreviewed} data-test-id="stack-trace">
+          {!newestFirst ? convertedFrames : [...convertedFrames].reverse()}
+        </Frames>
       </Wrapper>
     );
   }
 
   return (
-    <Wrapper isHoverPreviewed={isHoverPreviewed} data-test-id="stack-trace">
-      {!newestFirst ? convertedFrames : [...convertedFrames].reverse()}
+    <Wrapper className={className}>
+      <Frames isHoverPreviewed={isHoverPreviewed} data-test-id="stack-trace">
+        {!newestFirst ? convertedFrames : [...convertedFrames].reverse()}
+      </Frames>
     </Wrapper>
   );
 }
 
 export default Content;
 
-const Wrapper = styled('ul')<{isHoverPreviewed?: boolean}>`
+const Wrapper = styled('div')`
+  && {
```

Comment:
What does the double `&&` do again?

Reply:
it's like `!important` but more does not feel like 🙈

Reply:
https://dev.to/magischerzwerg/double-ampersand-trick-in-sass-with-react-4khe ah it changes the specificity by including the class twice in the selector 🤔